<launch>

  <!-- navigation common params-->
  <rosparam file="$(find run_task)/config/param_common/local_costmap_params.yaml"      command="load" ns="move_base"/>
  <rosparam file="$(find run_task)/config/param_common/global_costmap_params.yaml"     command="load" ns="move_base"/>
  <rosparam file="$(find run_task)/config/param_common/move_base_params.yaml"          command="load" ns="move_base"/>
  <rosparam file="$(find run_task)/config/param_common/base_global_planner_param.yaml" command="load" ns="move_base"/>
  
  <param name="move_base/base_local_planner"    type="string" value="teb_local_planner/TebLocalPlannerROS"/>

  <!-- Arguments参数 -->
  <arg name="car_mode"  default="" />

  <!-- car_mode and params-->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find run_task)/config/param_brushless_senior_diff/teb_local_planner_params.yaml"  command="load"/>
    <rosparam file="$(find run_task)/config/param_brushless_senior_diff/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find run_task)/config/param_brushless_senior_diff/costmap_common_params.yaml" command="load" ns="local_costmap" />
  </node>

</launch>


